### DISCUSSION TITLE
Firefly Shimmer 2.0.1

### DISCUSSION THREAD

**Discussion by Tuditi** [iotaledger/firefly#4908] [Announcements]
### Changelog

**Fixed**:
- Windows installations with missing dependencies
- Different decimal separators in deep links
- Large number support for relevant input UI components 
- Styling on some input UI components
- Styling on the network modal UI component
- Fix menu buttons for creating normal and developer profiles

**Added**:
- Alias activity in the Wallet dashboard
- Alias selection when minting native tokens
- Customized sorting for assets

**Changed**:
- Allow users to continue if no unclaimed rewards found

See the [full changelog](https://github.com/iotaledger/firefly/compare/desktop-2.0.0...desktop-2.0.1).
------
### File Hashes
[How to verify the authenticity of your Firefly Desktop download](https://wiki.iota.org/chrysalis-docs/firefly/verify_download)
| File | Platform | SHA256 Hash |
| --- | --- | --- |
| firefly-desktop-2.0.1.exe | Windows | 6843beb5d9f3d6ec7beeb1ceb1da3f4c978787ba2e66017b513baca54f0236b0 |
| firefly-desktop-2.0.1.AppImage | Linux | 949a393912e3b36af118301072b3340b3f9e8b7aaf17255d923e903f9d17007c |
| firefly-desktop-2.0.1.dmg | MacOS |  2fb1fb7a487f0a2a6813ecb41a9d703de1d1dfc45df72379344057df6fc1b6d0 |

<hr /><em>This discussion was created from the release <a href='https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.1'>Firefly Desktop 2.0.1</a>.</em>

**Comment by marzgithub858** [comment]
Hey there ,  all of the Firefly wallets I have downloaded so far have been working.  However , except for the Firefly shimmer desktop V2.0.1.  It just doesn't work in my pc laptop (windows 10, i5, RAM 6gb).  What is it that I'm missing?  does it require a particular task or a specific/higher spec with my hardware? I haven't claim my shimmer tokens up to now. 
Hope you guys can assist me on this concern. Thanks!

**Reply by JannemanDev** [reply]
What exactly doesnt work? The installation? Or after installation  
when you start the app you get a white screen for example?
Also do you use VPN?
The other working Fireflys was also installed on this machine and they do work? Are they still installed and working?
Do you have a working internet connection, also when setup is run?
Do you have some spare free disk space, a few hundred MB is already enough?

Always a good idea to verify the downloaded app so to be sure the correct non tampered setup has been downloaded. See release notes on howto.

You could also try to temporarely turn off any virusscanner for example Windows defender.

The only requirement/dependency is Microsoft VC++ 2015-2022 redistributable but this is automatically downloaded and installed during setup.

What bit OS do you use? 32 or 64bit? What language? Could you run systeminfo in cmd prompt and show the output?

**Reply by marzgithub858** [reply]
when I turn the app, all I get is a white screen. I tried to just turn it that way & let stay overnight but after 24hrs its just a white screen.
all other versions of firefly works just fine, I can even update it to newer versions. Just except this firefly shimmer desktop v2.0.1.

**Reply by marzgithub858** [reply]
here are 2 screenshots of my system info.

![systeminfoPNG](https://user-images.githubusercontent.com/70326803/198507738-39127968-9dc2-4892-937a-66758d33674c.PNG)
![systeminfo2](https://user-images.githubusercontent.com/70326803/198507739-d459ccc5-3aac-42c1-b0b9-aa6f239547fe.PNG)

**Reply by marzgithub858** [reply]
![systeminfo3](https://user-images.githubusercontent.com/70326803/198508469-32939e34-91f1-4c4f-9e90-b859576fcf8c.PNG)

**Reply by marzgithub858** [reply]
sorry, 3 screenshots.  the last one is after doing a "cleanmgr".

**Reply by marzgithub858** [reply]
What exactly doesnt work? The installation? Or after installation
 > after installation.

when you start the app you get a white screen for example?
 > Yes, a white screen.

Also do you use VPN?
 > I tried BOTH, using a VPN and without VPN, still didn't work.
 
The other working Fireflys was also installed on this machine and they do work? Are they still installed and working?
 > All the other Firefly from the 1st version to the latest works just fine.

Do you have a working internet connection, also when setup is run?
 > yes, I'm on a higher subscription internet fiber plan.

Do you have some spare free disk space, a few hundred MB is already enough?
 > Yes, I believe I have more than enough as you can see above screenshots.

Always a good idea to verify the downloaded app so to be sure the correct non tampered setup has been downloaded. See release notes on howto.

You could also try to temporarely turn off any virusscanner for example Windows defender.
 > Already turned OFF all antivirus apps, including the windows antivirus.

The only requirement/dependency is Microsoft VC++ 2015-2022 redistributable but this is automatically downloaded and installed during setup.
 > I think my pc laptop has more than adequate specs to handle it.

What bit OS do you use? 32 or 64bit? What language? Could you run systeminfo in cmd prompt and show the output?
 > I believe it's a 32bit as shown from system.info screenshot above.

**Comment by marzgithub858** [comment]
when I turn the app, all I get is a white screen.  I tried to just turn it that way & let stay overnight but after 24hrs its just a white screen.
all other versions of firefly works just fine, I can even update it to newer versions.  Just except this firefly shimmer desktop v2.0.1.

**Reply by Tuditi** [reply]
Just to make sure, can you check whether you can find Microsoft VC++ 2015-2022 in your installed programs?

**Reply by marzgithub858** [reply]
ok copy that. thanks mate 👍

**Reply by marzgithub858** [reply]
After downloading the Microsoft VC++ , the Firefly Shimmer v2.0.1 desktop app works right away. Thanks again mate!  
However,  after uploading the saved stronghold backup file/s , my $SMR tokens doesn't seem to show up in my wallet after a dozen of searches. But using the earlier version of Firefly1.3.0
 with the saved stronghold back up with , its right there as can be seen in the screenshot.  Am I missing something again here ? 

![SMRrewards](https://user-images.githubusercontent.com/70326803/198812917-4ac18dd3-8fa1-4587-9fb0-6142f47e2b53.PNG)

**Reply by JannemanDev** [reply]
There is a bug/limitation I think when restoring old versioned stringhold backups. Just use your seed.

Tuditi: we really need logging during installation to find out in this case why VC download(?) or install(?) failed.
Even better would be when this log file is auto uploaded to IF after installation if allowed by privacy or when explicitly given permission by user.

**Reply by marzgithub858** [reply]
I actually tried both logging via seedphrase and backup file, countless times still ""NO SHOW" of $SMR balance, no matter countles refresh/search.. Is there a bug that's currently being fixed ?

**Reply by Tuditi** [reply]
Are you running the claim reward flow or the search balance flow once you already restored your profile? You need to claim your tokens running the claim rewards flow. For more info see here: https://blog.shimmer.network/how-to-claim-smr-firefly/

**Reply by marzgithub858** [reply]
> Are you running the claim reward flow or the search balance flow once you already restored your profile? You need to claim your tokens running the claim rewards flow. For more info see here: https://blog.shimmer.network/how-to-claim-smr-firefly/

Hi Tuditi,
yes I've tried claiming rewards as per instructions as shown in the link you provided, probably more that a dozen times already.  Still unsuccessful up to now.

**Reply by marzgithub858** [reply]
here's the recent firefly-desktop.2.1.0 version I downloaded.  It says error every time I try to do the claim function.
Is this still an ongoing development and we need to wait for the fully functional version before we can start claiming our $SMR rewards ?

![image](https://user-images.githubusercontent.com/70326803/215377823-58732bf7-1917-403d-a6b2-ff46b002f438.png)

**Reply by Tuditi** [reply]
2.1.0 is still an alpha release and only connects to the test network, so you won't be able to claim your SMR rewards with it. If you made a lot of transactions on the Chrysalis network then it's possible that you need to run the claiming process 20+ times. I don't have any other advice until the new release is out.

**Comment by ehanuise** [comment]
Works well on linux.
One thing however : The name should reflect that this is a shimmer only wallet!
The Iota wallet and the shimmer wallet have the exact same name (Firefly Desktop) just different release versions (1.7.0 for Iota and 2.0.1 for Shimmer as I write this).
It took me a while to figure out that Firefly Desktop 2.0.1 is Shimmer only and does not replace 1.7.0, which is Iota only. Initial expectation was 2.0.1 replaced 1.7.0 and handled Iota and shimmer.
Furthermore when the Iota wallet advances, it could end up with the same version numbers as the shimmer wallet.
This is confusing as heck, and confusion is the last thing you want when dealing with a crypto wallet.
Please consider moving to a naming scheme that includes the token name, or use a different wallet name for Shimmer.

**Reply by Tuditi** [reply]
Thank you for reaching out! We are aware that the naming of the applications is not optimal at the moment. We will be deprecating the IOTA only wallet and provide one wallet with both Shimmer and IOTA support in the near future. That will solve these problems.

**Reply by ehanuise** [reply]
Great, a combined app for IOTA and related tokens (shimmer/assembly for the time being) is indeed much better than separate applications. Keep up the good work :)

**Comment by naxela57** [comment]
Hi all. I will try my best to explain my situation, but please let me know if it isn't clear.

Basically, I bought a new Mac and migrated everything from the old 2011 machine to the brand new one. When I tried to access my wallets in Firefly, I found that my PIN no longer worked, but have since found that this is not unusual when switching machines, as the PIN is specific to each machine. What I then did was to create a new wallet on my new machine and sent all of my IOTA to this new wallet. However, my Assembly and Shimmer rewards remain on the wallet that is only accessible on my old Mac.

I was aiming to use the Firefly Shimmer app to claim these rewards, but this app will not function on my old Mac, instead giving me a white screen. I have searched high and low for an answer, but am still stuck in this limbo of being unable to do anything with my Assembly and Shimmer rewards. I would like them to be present on the new machine's Firefly/Shimmer app/s, but no joy.

Is anyone able to assist me here? I'd really appreciate it.

Thanks in advance.

**Reply by Tuditi** [reply]
Hey @naxela57 , thanks for reaching out! Your staking rewards are tied to the addresses with which you staked. This means that if the staking period has ended and you moved your iotas to a new address you cannot claim the rewards from that address. 

To solve this you can create a Stronghold back-up from the original Firefly (v1) wallet you staked with on your old machine. Put the Stronghold file of that wallet onto your new machine and claim your Shimmer rewards in Firefly 2.0.1 using that file. If you have the mnemonic of that wallet, you can also use your mnemonic to achieve the same thing.

Let us know if it worked out!

**Comment by iapplemac** [comment]
Don’t(install) work on windows

**Reply by Tuditi** [reply]
Hey @freshMAC ,

We're a bit sad about your comment. Let us know what issue you encountered and we will work on fixing it. :relaxed:

Cheers! :relaxed:
